src/common/mod.rs

```diff
@@ -0,0 +1,39 @@
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+use thiserror::Error;
+
+/// Retrieve the current unix time in nanoseconds
+pub fn get_unix_time_millis() -> u128 {
+    let now = SystemTime::now();
+    let duration = now.duration_since(UNIX_EPOCH).unwrap_or_else(|_| {
+        println!("Failed computing UNIX timestamp during admin login!");
+        Duration::from_secs(0)
+    });
+    duration.as_millis()
+}
+/// Retrieve the current unix time in nanoseconds
+pub fn get_unix_time_seconds() -> u64 {
+    let now = SystemTime::now();
+    let duration = now.duration_since(UNIX_EPOCH).unwrap_or_else(|_| {
+        println!("Failed computing UNIX timestamp during admin login!");
+        Duration::from_secs(0)
+    });
+    duration.as_secs()
```

Comment:
```suggestion
    let duration = now.duration_since(UNIX_EPOCH).inspect_err(|_| println!("Failed computing UNIX timestamp during admin login!").unwrap_or_default().as_secs()
```

Reply:
Thanks for this, the print statement can be removed completely. Admin login does not make sense here. I think I copied this from one of my other projects.

Reply:
e889da77a756abf1d3390a98806640e31f58cfb7